selfdrive/modeld/modeld.cc

```diff
@@ -114,7 +111,7 @@ void run_model(ModelState &model, VisionIpcClient &vipc_client) {
 
       // tracked dropped frames
       uint32_t vipc_dropped_frames = extra.frame_id - last_vipc_frame_id - 1;
-      frames_dropped = (1. - frame_filter_k) * frames_dropped + frame_filter_k * (float)std::min(vipc_dropped_frames, 10U);
+      float frames_dropped = frame_dropped_filter.update((float)std::min(vipc_dropped_frames, 10U));
```

Comment:
The warmup is now different. You don't reset the internal filter state. You should probably add a `reset()` function to the filter for this.